src/Domain/Models/Leave.cs

```diff
@@ -19,5 +19,7 @@ public class Leave
         public string Description { get; set; }
         [Required]
         public string Status { get; set; }
+
+        public DaysTally? DaysTally { get; set; }
```

Comment:
This shouldn't be nullable to support our Domain. Remove `?` and add a new migration.